flexbox/src/main/res/values/attrs.xml

```diff
@@ -99,5 +99,8 @@ limitations under the License.
             <enum name="baseline" value="3" />
             <enum name="stretch" value="4" />
         </attr>
+
+        <attr name="layout_minWidth" format="dimension" />
```

Comment:
Also could you update the FlexItem class under the app module?
Otherwise layout_minWidth/layout_minHeight aren't restored after a configuration change for the demo app.